test/partition.test.js

```diff
@@ -0,0 +1,19 @@
+const { partition } = require("../lib");
+
+describe("partition", () => {
+  test("should partition array slices of given length", () => {
+    expect(typeof partition([1, 2, 3, 4], 1)).toEqual([[1], [2], [3], [4]]);
```

Comment:
Hey @SamHowie ! Thanks for your contributions. I'm a little confused on the test methods here. It looks like you are expecting a `typeof` return to equal actual values as opposed to a data type?

Reply:
Oh you are right this was a silly error on my part. Thanks for catching it!

Fix incoming...

Reply:
Thanks !